ui/src/utils/lineWriter.ts

```diff
@@ -0,0 +1,129 @@
+import AJAX from 'src/utils/ajax'
```

Comment:
Is there a reason we're using `AJAX` rather than `fetch`?

Reply:
AJAX uses axios, which uses XMLHTTPRequest which is more widely supported

Reply:
TIL

Reply:
we polyfill fetch though right? since we are actively depreciating anything that uses axios (as it's currently only tied to the old api client), it would be cool to see new code not using it so that we can keep api auth mechanisms centralized in `src/client` (like where tokens are stored and header params and the like). Not a blocker, just bumps the work to the client library tech debt epic right off the bat.

Reply:
@drdelambre my thinking is since this isn't a client interacting with our api, but making a request to an external service, it shouldn't really go in `src/client`. i was under the assumption that `src/client` is for communicating with our api via swagger. is that not the case? if it is, do we have a story around how to make requests to external services or resources? i assumed that was the intent of `AJAX`.

either way, first I'd heard we're actively deprecating anything that uses axios. an effective way to let people know this is to put a comment saying something is deprecated and shouldn't be used and link to a place that explains that. do we have any documentation like an rfc around deprecating axios that i can link to? or anything explaining what to use instead?

Reply:
we do indeed polyfill fetch.  would be nice not to rely on axios for this.  this def doesn't need to (or should be) within `src/client`.  i agree that we should just rip axois out altogether and eliminate the confusion.

Reply:
We had a big talk about it during planning a month ago when bonitoo was releasing the new api client, and made these in an effort to put a bow on an effort that was 90% of the way there:
https://github.com/influxdata/influxdb/issues?q=is%3Aopen+is%3Aissue+label%3Ateam%2Fmonitoring+label%3Akind%2Ftech-debt+generated+client
Everyone has been busy doing other stuff though. We currently treat our queries as an external resource (weird right?) within the app, which you can get a gander of over here:
https://github.com/influxdata/influxdb/blob/master/ui/src/shared/apis/query.ts#L55